Gemfile.lock

```diff
@@ -40,7 +38,7 @@ DEPENDENCIES
   unicorn
 
 RUBY VERSION
-   ruby 2.6.8p205
+   ruby 3.3.0p0
```

Comment:
@tcdowney and @gururajsh - I saw in the original pr that this said ruby 3.3 but everywhere else it says 3.2.  I think it would be good to have this be consistent with one of the versions rather than both

Reply:
Good catch, fix: https://github.com/cloudfoundry/CLAW/pull/80

I had rebundled, but I guess it won't downgrade it unless you explicitly do `bundle update --ruby`.